app/controllers/orders_controller.rb

```diff
@@ -15,6 +15,10 @@ def create
       )
       add_items_to_order
       flash[:order_success] = {color: 'green', message: "Order was successfully placed"}
+      if @user.total_purchased >= 50000
+        flash_link = "#{view_context.link_to 'Doomsday', doomsday_path}"
+        flash[:alert] = {color: 'red', message: "Doomsday Hath Commeth: #{flash_link} "}
```

Comment:
Line is too long. [88/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.